www/partial/editor.html

```diff
@@ -15,6 +15,13 @@
 		<!--<form action="/download" method="post">-->
 		<!--<input type="hidden" name="data" ng-value="pro5Ctrl.parsed">-->
 		<button ng-click="pro5Ctrl.download()">Download {{pro5Ctrl.templateExt}} file</button>
+		<div id="checkbox-container">
+			<input ng-model="referenceCaps" ng-change="pro5Ctrl.referenceChangeCaps(referenceCaps)" type="checkbox" name="reference"></input>
+			<label for="reference">Reference Caps</label>
```

Comment:
These labels are not clickable, see https://pp-gen-uppercase-checkb-zlkpiy.herokuapp.com/#/editor/hillsongstockholm/
`<input>` needs the same id as the label's for attribute.
Otherwise, do something like `<label><input ..> <span>Reference Caps</span></label>` to automatically connect the label with the input